spec/acceptance/00_meta_spec.rb

```diff
@@ -14,7 +14,7 @@
 describe 'logstash module' do
   it 'should be available' do
     shell(
-      "ls #{default['distmoduledir']}/logstash/Modulefile",
+      "ls #{default['distmoduledir']}/logstash/metadata.json",
```

Comment:
Thank you! I can't believe I missed this.

It goes to show that there is no such thing as a "trivial" change.